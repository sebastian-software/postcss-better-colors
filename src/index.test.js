import postcss from "postcss"

import plugin from "."

const options = { from: undefined }

test("Transform Red", () => postcss([ plugin ])
  .process(".red { color: red; }", options)
  .then((result) => {
    expect(result.css).toMatchSnapshot()
  }))

test("Transform Blue", () => postcss([ plugin ])
  .process(".blue { color: blue; }", options)
  .then((result) => {
    expect(result.css).toMatchSnapshot()
  }))

test("Transform Green", () => postcss([ plugin ])
  .process(".green { color: green; }", options)
  .then((result) => {
    expect(result.css).toMatchSnapshot()
  }))

test("Transform Orange", () => postcss([ plugin ])
  .process(".orange { color: orange; }", options)
  .then((result) => {
    expect(result.css).toMatchSnapshot()
  }))

test("Transform Yellow", () => postcss([ plugin ])
  .process(".yellow { color: yellow; }", options)
  .then((result) => {
    expect(result.css).toMatchSnapshot()
  }))
