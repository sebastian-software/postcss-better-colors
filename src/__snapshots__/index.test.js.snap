// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transform Colors 1`] = `".red { color: #FF4136; }"`;
